librz/analysis/analysis.c

```diff
@@ -154,6 +155,9 @@ RZ_API RzAnalysis *rz_analysis_free(RzAnalysis *a) {
 	if (!a) {
 		return NULL;
 	}
+
+	plugin_fini(a);
```

Comment:
what about all other non-current analysis plugins? Shouldn't we `fini` them as well?

Reply:
only if they have static context, yes definitely